bot/internal/bot/bot.go

```diff
@@ -146,7 +159,17 @@ func isLargePR(files []github.PullRequestFile) bool {
 		additions += f.Additions
 		deletions += f.Deletions
 	}
-	return additions-deletions > 1500
+	delta := additions - deletions
+	switch {
+	case delta < 500:
```

Comment:
Nit: Maybe < 100 for small? It just "feels" like anything over 100 characters is not a trivial changeset.

Reply:
Sure, totally open to changing the sizes. (BTW, these are _lines_, not characters)

My goal for this was more to convey to a reviewer when they open a PR how much they have to review. IMO, "small" should mean "I can review this in 10 minutes or less." Do you think 100 is the right number for that?

Reply:
I think it depends on the reviewer tbh. For me personally, 500 lines feels like something I should carve out time for and not an easy fix. 100 feels like something that can be reviewed quickly. We can always experiment and tweak the numbers.